kolibri/core/assets/src/views/SyncStatusDescription.vue

```diff
@@ -53,6 +54,11 @@
         message: 'Device isn’t connected to a server it can sync with',
         context: 'Device syncing status explained.',
       },
+      insufficientStorageDescription: {
+        message:
+          'This device does not have enough storage for updates. Check the size of your visible lessons and quizzes. Making a lesson or quiz not visible will remove it from learner devices.',
+        context: 'Device syncing status explained.',
```

Comment:
I would suggest rephrasing this and the above one to:

`Description of the device syncing status.`